add termux (#257)
vlang/vab: README.md

E.g.: `sudo apt install openjdk-8-jdk`

### termux (experimental)

> You must install java-17 and few more thing:

Reviewer: ```suggestion
You must install Java 17 and few more things:
```

```bash
pkg install openjdk-17 aapt apksigner dx ecj
```
Download, unzip and set enviroment variable to sdk and ndk from https://github.com/Lzhiyong/termux-ndk/releases 

> you maybe will have to set sdk version into compatible with ndk (`sdkmanager install/uninstall platform-version`) 

Reviewer: Keeping lines below 100 chars in markdown.md files will keep the CI checks happy:
```suggestion
you will maybe have to set the SDK version to be compatible with the NDK
(`sdkmanager install/uninstall platform-version`) 
```

> and enjoy use vab !

Reviewer: ```suggestion
and enjoy using vab!
```
